host_vars/deploy/main.yml

```diff
@@ -47,7 +47,7 @@ ngi_resources: "{{ root_path }}/resources/"
 sarek_tag: "3.4.2"
 sarek_dest: "{{ sw_path }}/sarek/{{ sarek_tag | replace('.', '_') }}"
 
-rnaseq_tag: "3.12.0"
+rnaseq_tag: "v3.12.0_ngi_v1-rc1"
 rnaseq_dest: "{{ sw_path }}/rnaseq/{{ rnaseq_tag | replace('.', '_') }}"
```

Comment:
What is the purpose of the `rnaseq_dest`? It uses the updated `rnaseq_tag`, but in `roles/nf-core/tasks/pipeline.yml` the outdir is set to `--outdir {{ sw_path }}/{{ pipeline }}`. 

I could not find an additional moving/copying step, so in that case it would not be obvious in the file system, that we are using an modified pipeline version instead of the native nf-core version, no?

Reply:
No. I recalled that an explicitly specified `--outdir` would override whatever automatic folder name is constructed from the pipeline name and revision, if that argument is omitted. But I just tested it manually, and you are correct - there are subfolders named according to the revision. Probably to enable multi-revision downloads with the classic method, that is without the `-p` / `--platform` flag.